t/endpoint.t

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env perl
+
+use 5.028;
+use warnings;
+use experimental qw(signatures);
+
+use Test::Lib;
+use Test::PromAlertProxy;
+use Test::More;
+use Test::Time time => Test::PromAlertProxy->now;
+
+use IO::Async::Loop;
+
+use Plack::Test;
+use HTTP::Request::Common;
+use JSON::MaybeXS;
+
+use PromAlertProxy::Hub;
+
+use PromAlertProxy::Logger '$Logger' => { init => {
+  ident    => 'promalertproxy',
+  to_self  => 1,
+} };
+
+my $loop = IO::Async::Loop->new;
+
+my $hub = PromAlertProxy::Hub->new(loop => $loop);
+
+my $target = Test::PromAlertProxy::Target->new(
+  hub     => $hub,
+  id      => 'test',
+  default => 1,
+);
+$hub->add_target($target);
+
+my $t = Plack::Test->create($hub->_app);
+
+my $res = $t->request(
+  POST '/api/v2/alerts',
+    'Content-type' => 'application/json',
+    Content => encode_json([ Test::PromAlertProxy->prom_alert ]),
+);
+
+ok($res->is_success, "prom alert successfully posted");
+
+$loop->loop_once;
+
+is(scalar $target->received_alerts->@*, 1, 'alert was dispatched to target');
```

Comment:
So it seems! I did it because I expected an array to be exploded out into the args, but you are indeed right, `is()` has a `$` prototype for its first arg. This is not documented though.